### DISCUSSION TITLE
Welcome to open-iscsi Discussions!

### DISCUSSION THREAD

**Discussion by gonzoleeman** [open-iscsi/open-iscsi#346] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wenchao-hao** [comment]
Hi Lee, it seems too late to join this discussions. Here is my self introduction:

I am fresh man in iSCSI we package open-iscsi and supply it to our customers. I am now focus on some abnormal scenario such as iscsid died and restart in any time(Although this is a low probability event, I think we should correctly handle this scenario). What's more I am using open-iscsi in initrd or with isns.

I found some issues when iscsid died durning login/logout and trying to fix it, I would submit PRs when there are done.

**Reply by gonzoleeman** [reply]
And I personally really appreciate your contributions! We need more engineers like you. Thank you!
